### ISSUE TITLE
Android starts with empty activity than switches to the full app

### ISSUE THREAD

**Issue by rborn** [wix/react-native-navigation#522]
So just what the title says, the app opens with an empty activity and immediately switches to a new one? 😄 

![wixnav](https://cloud.githubusercontent.com/assets/406409/20796279/c7fe2fb2-b7d5-11e6-94b1-39d0e9444148.gif)

---
### Environment
* React Native Navigation version: 2.0.0-experimental.150
* React Native version: 0.38.0
* Platform(s) Android
* Device info (Simulator/Device? OS version? Debug/Release?): Genymotion, 5.1.0

**Comment by guyca** [comment]
The first "empty" activity you see is your MainActivity which extends SplashActivity. While it's displayed, react context is loading in the background and when it's ready your actual application is displayed in NavigationActivity.

You can display a splash screen while MainActivity is visible.

We might abandon the two activity pattern in favor of a single activity in the future but that won't happen any time soon.

**Comment by rborn** [comment]
Can we change the animation of the second activity (to none or fade or whatever)? The app loads too fast (at least in this case) to make the splash screen really useful (adding a delay would be bad UX too)

Sorry, kind of noob here with RN 😄

**Comment by guyca** [comment]
You can disable the animation by specifying `animationType: none` when [starting your app](https://github.com/wix/react-native-navigation/wiki/Top-Level-API#starttabbasedappparams)

**Comment by rborn** [comment]
Mmm, it looks like somebody has to read the docs 😊 

Thank you!

**Comment by florianbepunkt** [comment]
@guyca for me setting `animationType: 'none'` in `startSingleScreenApp` or `startTabBasedApp` has no effect… the second activity (afer splash screen) slides in regardless of the setting.

**Comment by ferrannp** [comment]
@guyca,

> We might abandon the two activity pattern in favor of a single activity in the future but that won't happen any time soon.

I feel this is quite an issue. The splash screen is not really a pattern that Android apps use, actually, as an Android user I found quite annoying apps with splash screen at the beginning, to see it once is ok, but every time I want to open the app... Makes me go slower. 

But in this case, we are forced to use a splash screen in our apps for not having that "empty activity" feeling, but it is also so fast that it makes the flow to be weird. Furthermore, when you rotate the device, it is also showing an empty activity first and then it starts a second one... So it's like, we would need a splash screen for landscape and it will also be shown every time we rotate the device.

In summary, if the same can be done with a single activity, I think this is a must for Android.

**Comment by jslz** [comment]
I tried animationType:'none' but that didn't fix it for me. From looking at other issues here, it seems like this doesn't work in v1.x but is addressed in v2.x which is still experimental/unreleased?
